src/main/java/pl/javastart/task/ArrayUtils.java

```diff
@@ -0,0 +1,41 @@
+package pl.javastart.task;
+
+import java.util.Scanner;
+
+class ArrayUtils {
+    private static final Scanner INPUT = new Scanner(System.in);
+
+    double[] createFloatArray() {
+        System.out.println("Podaj rozmiar tablicy liczb zmiennoprzecinkowych:");
+        int arraySize = 0;
+        while (!(arraySize > 0)) {
+            arraySize = INPUT.nextInt();
+            if (arraySize < 0) {
+                System.out.println("Podano ujemny rozmiar tablicy");
+            } else if (arraySize == 0) {
+                System.out.println("Podano zerowy rozmiar tablicy");
+            }
+        }
```

Comment:
Jeżeli pętla z założenia powinna wykonać się co najmniej raz, to lepiej używać w takim przypadku do-while